llvm/test/CodeGen/AMDGPU/cgp-addressing-modes.ll

```diff
@@ -166,47 +166,6 @@ done:
   ret void
 }
 
-; This ends up not fitting due to the reserved 4 bytes at offset 0
```

Comment:
Since we don't have the reserved bytes anymore, I think we can skip this test (there's no special behavior compared to the other tests).

Reply:
I would keep the test anyway to show the change in behavior